opencga-core/src/main/java/org/opencb/opencga/core/models/clinical/pipeline/genomics/GenomicsAlignmentPipelineTool.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015-2020 OpenCB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.opencb.opencga.core.models.clinical.pipeline.genomics;
+
+import org.opencb.commons.annotations.DataField;
+import org.opencb.opencga.core.models.clinical.pipeline.PipelineTool;
+
+import java.util.Map;
+
+public class GenomicsAlignmentPipelineTool extends PipelineTool {
+
+    @DataField(id = "index", description = "Tool index path")
+    private String index;
```

Comment:
index as "tool index path" ? What would be the expected value here? A path from the docker?